code/modules/mob/living/carbon/brain/brain_life.dm

```diff
@@ -1,20 +0,0 @@
-/mob/living/carbon/brain/proc/handle_temperature_damage(body_part, exposed_temperature, exposed_intensity)
-	if(status_flags & GODMODE)
-		return
-
-	if(exposed_temperature > bodytemperature)
-		var/discomfort = min( abs(exposed_temperature - bodytemperature)*(exposed_intensity)/2000000, 1.0)
-		adjustFireLoss(20.0*discomfort)
-
-	else
-		var/discomfort = min( abs(exposed_temperature - bodytemperature)*(exposed_intensity)/2000000, 1.0)
-		adjustFireLoss(5.0*discomfort)
```

Comment:
Can you mention this in your PR too? I cant seem to find it being mentioned anywhere else, otherwise LGTM

Reply:
This proc wasn't ever called from what I can tell. Note how it isn't even an override.